### PR TITLE
Add aliases for core properties, and fix update options

### PR DESCRIPTION
**Pull request by LuKks**
If you're using a Hyperbee then there should be no need to constantly access to `db.core`

Yes, they're just core aliases but makes the bee experience better

Closes https://github.com/holepunchto/hyperbee/issues/128

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -346,8 +346,8 @@ class Hyperbee extends ReadyResource {
     return this.core.replicate(isInitiator, opts)
   }
 
-  update () {
-    return this.core.update({ ifAvailable: true, hash: false })
+  update (opts) {
+    return this.core.update({ ...opts, ifAvailable: true, hash: false })
```
Comment (LuKks) [inline index.js:350]: Should I remove `ifAvailable` and `hash`?

Comment (mafintosh) [reply index.js:350]: ya just pass opts

**Comment by LuKks** [comment]
@mafintosh Ready!

**Comment by LuKks** [comment]
Fixes https://github.com/holepunchto/hyperbee/issues/128

**Pull request merged**